randovania/games/super_metroid/logic_database/header.json

```diff
@@ -350,6 +350,11 @@
                 "long_name": "X-Ray Climb",
                 "description": "While stuck in a wall, Samus can cancel a crouched turnaround by releasing the X-Ray Scope which forces her to stand. This gains her a little height and can be repated ad infinitum.",
                 "extra": {}
+            },
+            "SnailClimb": {
+                "long_name": "Snail Climb",
+                "description": "By manipulating the snails in certain areas of Maridia, Samus can use them to climb to heights otherwise unreachable.",
```

Comment:
How many instances do you think there will be of this trick (and also do they have varying difficulty)? am considering whether it's worth to have as a seperate one.

Reply:
There are 4 that I can think of. And the one to the missile pack is considerably more difficult. And in general, the further you have to go the harder I would rate it. 
![image](https://github.com/randovania/randovania/assets/46821362/1d91bf83-3136-4a6c-8477-e64486326174)

Reply:
are they only in this room?

Reply:
Yeah

Reply:
hm. definitely feels weird to have a trick for only one room. feel like it might be better to just use varying degrees of movement instead if it's onily for 4 examples.

Reply:
changed to movement and removed trick from header